LEGO1/lego/legoomni/src/actors/act2actor.cpp

```diff
@@ -82,19 +95,165 @@ void Act2Actor::SetWorldSpeed(MxFloat p_worldSpeed)
 	m_unk0x44 = 0;
 }
 
-// STUB: LEGO1 0x10019520
-// STUB: BETA10 0x1000d4d6
+// FUNCTION: LEGO1 0x100192a0
+// FUNCTION: BETA10 0x1000d4d6
+void Act2Actor::FUN_100192a0(undefined4 p_param)
+{
+	Mx3DPointFloat local38(0.0, 0.0, 0.0);
+	Mx3DPointFloat local4c(0.0, 0.0, 0.0);
+
+	if (m_grec) {
+		delete m_grec;
+	}
+
+	m_grec = new LegoPathEdgeContainer();
+	assert(m_grec);
+
+	local38 = Vector3(g_unk0x100f0db8[p_param].m_unk0x00);
+	local4c = Vector3(g_unk0x100f0db8[p_param].m_unk0x0c);
+
+	LegoPathBoundary* otherBoundary = m_controller->GetPathBoundary(g_unk0x100f0db8[p_param].m_unk0x18);
+
+	MxResult sts = m_controller->FUN_10048310(
+		m_grec,
+		Vector3(m_roi->GetWorldPosition()),
```

Comment:
We should remove the explicit call to `Vector3` here - the compiler will do it implicitly